src/pages/get-eth.js

```diff
@@ -331,14 +331,15 @@ const GetETHPage = ({ data }) => {
               <Translation id="page-get-eth-." />
             </em>
           </p>
+          <br />
```

Comment:
Perhaps we add margin-top vs. a `<br/>`? They can act a bit unpredictably.

Reply:
done.